credentials-generator/src/main/resources/application.properties

```diff
@@ -0,0 +1,25 @@
+#
+# ====================
+# DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+#
+# Copyright (c) 2014 ForgeRock AS. All Rights Reserved
+#
+# The contents of this file are subject to the terms of the Common Development
+# and Distribution License("CDDL") (the "License").  You may not use this file
+# except in compliance with the License.
+#
+# You can obtain a copy of the License at
+# http://opensource.org/licenses/cddl1.php
+# See the License for the specific language governing permissions and limitations
+# under the License.
+#
+# When distributing the Covered Code, include this CDDL Header Notice in each file
+# and include the License file at http://opensource.org/licenses/cddl1.php.
+# If applicable, add the following below this CDDL Header, with the fields
+# enclosed by brackets [] replaced by your own identifying information:
+# "Portions Copyrighted [year] [name of copyright owner]"
+# ====================
+# Portions Copyrighted 2016 ConnId
```

Comment:
Remove

Reply:
As far as I understood we need a license like [this](https://github.com/andrea-patricelli/ConnIdGoogleAppsBundle/blob/GOOGLEAPPS-12/src/main/java/net/tirasa/connid/bundles/googleapps/GoogleAppsCustomSchema.java#L1-L15) since are new files

Reply:
Correct.